File: Projects/TransportathonHackathon.API/TransportathonHackathon.Application/Features/TransportRequests/Queries/GetById/GetByIdTransportRequestResponse.cs
﻿namespace TransportathonHackathon.Application.Features.TransportRequests.Queries.GetById
{
    public class GetByIdTransportRequestResponse
    {
        public Guid Id { get; set; }
        public string CustomerFirstName { get; set; }
        public string CustomerLastName { get; set; }
        public string CompanyName { get; set; }
        public string CountryFrom { get; set; }
        public string CountryTo { get; set; }
        public string CityFrom { get; set; }
        public string CityTo { get; set; }
        public bool IsOfficce { get; set; }
        public string PlaceSize { get; set; }
        public DateTime CreatedDate { get; set; }
        public DateTime UpdatedDate { get; set; }
        public DateTime StartDate { get; set; }
        public DateTime? FinishDate { get; set; }
    }
}

File: Projects/TransportathonHackathon.Clients/angular-ui/src/app/models/response-models/transport-requests/getByIdTransportRequestResponse.ts
export interface GetByIdTransportRequestResponse {
  id: string;
  transportTypeId: string;
  transportType: string;
  customerFirstName: string;
  customerLastName: string;
  companyName: string;
  countryFrom: string;
  countryTo: string;
  cityFrom: string;
  cityTo: string;
  isFinished: boolean;
  placeSize: string;
  createdDate: Date;
  updatedDate: Date;
  startDate: Date;
  finishDate: Date | null;
}
